src/constants/chains.tsx

```diff
@@ -22,17 +23,27 @@ export function getKeyChainByValue(chainId: ChainsValues) {
   return Object.keys(Chains).find((key) => Chains[key as ChainsKeys] === chainId)
 }
 
+const toHex = (chainId: number) => {
+  return '0x' + chainId.toString(16)
+}
+
 export type ChainConfig = {
   blockExplorerUrls: string[]
   chainId: ChainsValues
   chainIdHex: string
   icon?: React.ReactNode
+  nativeCurrency: {
+    name: string
+    symbol: string
+    decimals: number
+  }
   isL2?: boolean
   iconUrls: string[]
   id: ChainsValues
   isProd: boolean
   name: string
   rpcUrl: string
+  defaultRpcUrl: string
```

Comment:
Do we need both rpcUrl and defaultRpcUrl properties?

Reply:
I would say yes, the rpcUrl is the Aelin rpc url and the default rpc url is a public rpc, and the last one, it's only going to be used when a user doesn't have the network added, if we someone add a network from the dapp, we should add this but using the public rpc. Similar to what https://chainlist.org/ does